plugins/SiegeWar/config.yml

```diff
@@ -678,18 +678,18 @@ trap_warfare_mitigation:
   # TIP: 
   # If this feature is enabled, 
   # then it is recommended to also have a server rule requiring attackers to remove all traps in the area before the banner is placed
-  enabled: 'false'
+  enabled: 'true'
```

Comment:
Keep this disabled. Equinox had this feature and it just felt stupid that you couldn't mine around the banner. Anyone cloning this repo can simply change it